### PR TITLE
Creating Private Notices Notification view

### PR DESCRIPTION
**Pull request by shriyanka**

### CODE REVIEW

File: private_notices/views.py [hunk 1]
```diff
@@ -23,3 +24,18 @@ def post(self, request):
...
 		return "form"
 '''
+
+class NotificationView(APIView):
+	permission_classes = (IsAuthenticated,IsPrivateNoticeOwner,)
+
+	def get(self, request, user_id):
+		try:
+			user = User.objects.get(id=user_id)
+			print user.username
```
Comment (DESHRAJ) [inline private_notices/views.py]: @shriyanka can you remove the debug statement :stuck_out_tongue:

File: private_notices/serializers.py [hunk 2]
```diff
@@ -9,10 +9,11 @@ class NotificationSerializer(serializers.Serializer):
 
 
 class PrivateNoticeViewSerializer(serializers.ModelSerializer):
-    notification = NotificationSerializer(many=True)
-    sender = serializers.SerializerMethodField('return_sender')
+    pnotice = serializers.CharField(max_length=500)
```
Comment (DESHRAJ) [inline private_notices/serializers.py]: lets rename the variable `pnotice` with `private_notice` to make it more verbose.

File: private_notices/views.py (hunk 1 above)
Comment (DESHRAJ) [inline private_notices/views.py]: @shriyanka I am not sure why you have used rest_framework for the response. I think we are not creating APIs now.

**Comment by shriyanka** [comment]
@DESHRAJ  This is just for now for my work speed. later will return a context. :smile:

File: private_notices/serializers.py (hunk 2 above)
Comment (shriyanka) [inline private_notices/serializers.py]: Ok..

File: private_notices/views.py (hunk 1 above)
Comment (shriyanka) [inline private_notices/views.py]: @DESHRAJ  Nopes. Debug statements are helpful :smile: 

PS: will remove it no worries :smile:

**Comment by DESHRAJ** [comment]
Lets close this as sending private notices has already been integrated as model fields of `Notice` model

**Pull request closed without merging**